docs/grpc_disruption/setup_instructions.md

```diff
@@ -2,5 +2,34 @@
 
 Run `brew install protobuf` or `make install-protobuf-macos`
 
-# Running Chaos Dogfood Server & Client in order to test gRPC disruption
-Coming soon (once inital PR is merged)
\ No newline at end of file
+# Running chaos dogfood server & client in order to test gRPC disruption
+
+To locally run the dogfood server, call `go run grpcdogfood/server/dogfood_server.go` from the root directory of this project.
+
+Expected output:
+```
+connecting to localhost:50051...
```

Comment:
i'd expect the server to listen and the client to connect? but this sample output is the opposite

Reply:
Oh shoot, thank you for the catch. I updated and added some final explanation of how the server should respond to a client connecting.